pipelines/py/bundle.yaml

```diff
@@ -0,0 +1,36 @@
+name: Install Python packages with UV
+
+needs:
+  packages:
+    - python-${{inputs.python-version}}-base
+    - uv
+
+inputs:
+  dependencies:
+    description: "Indirect dependencies to install."
+    default: ""
+  package:
+    description: "Name of packages to install."
```

Comment:
I expect we'd normally want the full pin syntax. What if we allowed passing in a requirements file and/or lockfile instead?

Reply:
Yeah, I suppose ultimately we want to support both. It would be annoying to have to generate a requirements file if you just need a single package. But there are a number of packages that will benefit from `requirements.txt` support.